packages/editor/src/translation/process-editor/de.json

```diff
@@ -1,20 +1,32 @@
 {
   "a11y": {
+    "autocomplete": {
+      "noSuggestions": "Keine Vorschläge"
+    },
+    "diagram": {
+      "label": "Diagramm"
+    },
+    "grid": {
+      "message": "Wähle eine Ziffer von 1-9 um in den jeweiligen Bereich hinein zu zoomen."
+    },
```

Comment:
then we also do not need this translation here right?

Reply:
You are correct, we no longer need this and I removed it.